Cargo.toml

```diff
@@ -13,63 +13,10 @@ members = [
     "common",
     "common/codegen",
     "core",
-    "core-backend/codegen",
-    "core-backend/common",
-    "core-backend/sandbox",
-    "core-backend/wasmi",
+    "core-backend/*",
     "core-processor",
     "core-errors",
-    "examples/async",
-    "examples/async-custom-entry",
-    "examples/async-init",
-    "examples/async-signal-entry",
-    "examples/async-tester",
-    "examples/backend-error",
-    "examples/btree",
-    "examples/calc-hash",
-    "examples/capacitor",
-    "examples/compose",
-    "examples/constructor",
-    "examples/delayed-sender",
-    "examples/distributor",
-    "examples/fungible-token",
-    "examples/stack-allocations",
-    "examples/futures-unordered",
-    "examples/gas-burned",
-    "examples/incomplete-async-payloads",
-    "examples/init-fail-sender",
-    "examples/init-wait",
-    "examples/init-wait-reply-exit",
-    "examples/messager",
-    "examples/mul-by-const",
-    "examples/ncompose",
-    "examples/new-meta",
-    "examples/node",
-    "examples/out-of-memory",
-    "examples/piggy-bank",
-    "examples/ping",
-    "examples/program-factory",
-    "examples/program-generator",
-    "examples/proxy",
-    "examples/proxy-relay",
-    "examples/proxy-reservation-with-gas",
-    "examples/read-big-state",
-    "examples/reservation-manager",
-    "examples/reserve-gas",
-    "examples/rwlock",
-    "examples/send-from-reservation",
-    "examples/signal-entry",
-    "examples/state-rollback",
-    "examples/sync-duplicate",
-    "examples/sys-calls",
-    "examples/syscall-error",
-    "examples/vec",
-    "examples/wait",
-    "examples/wait-timeout",
-    "examples/wait_wake",
-    "examples/waiter",
-    "examples/waiting-proxy",
-    "examples/wat",
+    "examples/[!big-wasm]*",
```

Comment:
We had trouble with wildcards in `examples` some time ago. When you have some new example in a branch, you get a compilation error when returning to the master branch as you have a new example directory (with `.binpath`), but no manifest inside it.

Reply:
mb we can  `examples/[!big-wasm|!.binpath]` when meet that problem?